### ISSUE TITLE
[Question] How can I translate these fields?

### ISSUE THREAD

**Issue by adhonay** [formio/angular#945]
![image](https://user-images.githubusercontent.com/14100568/225599332-0728714a-794a-4e79-ad6f-b79587ac810d.png)
![image](https://user-images.githubusercontent.com/14100568/225599479-5deb91bd-6725-47e2-8955-853f852e9e1d.png)

These values ​​are inside a placeholder of the fields, but I can't translate, all others I managed like:

Rendered field, search fields and default value (inside edit modal)
```
<input placeholder="Search field(s)" ref="sidebar-search" type="search" class="form-control builder-sidebar_search">

<input pinputtext="" type="text" formcontrolname="text" class="p-inputtext p-component p-element ng-untouched ng-pristine ng-valid ng-star-inserted" ng-reflect-name="text" placeholder="Default Value">
```

How am I translating:

```
export const ptBrFormBuilder = {
    'No Matches Found': 'Nenhum encontrado',
    'Drag and Drop a form component': 'Arrastar e solte um componente de formulário',
    "Loading...": "Carregando...",
    "searchFields": "Campo(s) de pesquisa", // not working
    'Search field(s)': 'Campo(s) de pesquisa',// not working
    'Default Value': 'Valor padrão'//not working
};
};
```

**Comment by brendanbond** [comment]
Those values are themselves form.io form JSON definitions, see here: https://github.com/formio/formio.js/blob/c9311f87fcd18ee1b75030cde921b8d6b97cf4ce/src/components/_classes/component/editForm/Component.edit.data.js#L14

**Comment by adhonay** [comment]
> Those values are themselves form.io form JSON definitions, see here: https://github.com/formio/formio.js/blob/c9311f87fcd18ee1b75030cde921b8d6b97cf4ce/src/components/_classes/component/editForm/Component.edit.data.js#L14

Again I think you shouldn't conclude the post without actually answering the question asked, it's disrespectful.

I was able to translate globally as the documentation explains all fields, except those two I mentioned above.

However, all my components in the defautValue field do not translate this field, even passing the translation json.

And modifying the editForm of all components manually to translate the placeholder is not feasible.

And it doesn't show anywhere how to translate the builder especially the **Search field(s)** and how to hide it.

The text of the form Builder '**drag and drop a form component**' I was able to translate normally, but the text **Seach Field(s)** I can't through the translation json, maybe there is a specific way that is not documented.
